### ISSUE TITLE
Syntax error with alter table adding multiple columns

### ISSUE THREAD

**Issue by jackhftang** [scylladb/scylla#5263]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [✓] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): scylladb/scylla:3.0.10,  scylladb/scylla:3.1.1
Cluster size: 3
Platform (physical/VM/cloud instance type/docker): docker

*Test CQL*
```
create table test_alter( a int primary key) ;
alter table test_alter add b int; 
alter table test_alter add c int , d int;
```
*Output*
```
SyntaxException: line 1:33  : syntax error...
```

According to the syntax defined [here](https://docs.scylladb.com/getting-started/ddl/#alter-table-statement), the above cql should be supported. line 1:33 corresponds to the comma and I cannot figure out what actually go wrong with the comma, unless multi-column addition is not supported.

**Comment by amoskong** [comment]
Hi @jackhftang  The correct CQL statement is:  `alter table test_alter add (c int, d int);`

**Comment by tzach** [comment]
Closing, as far as I can see the doc is correct.
An example might help. @lauranovich FYI

**Comment by jackhftang** [comment]
@amoskong I tried `alter table test_alter add (c int, d int);`  the output is still syntax error `SyntaxException: line 1:27 no viable alternative at input '('`

**Comment by amoskong** [comment]
It works well with 3.1.1-0.20191024.f32ec885c.  
```
cqlsh> use keyspace1;
cqlsh:keyspace1> alter table test_alter add (b1 int, b2 int);
cqlsh:keyspace1> 
```

But It doesn't work with scylla 3.0.10  (3.0.10-0.20190815.b3bfd8c08)
```
cqlsh:keyspace1> alter table test_alter add a1 int;
cqlsh:keyspace1> alter table test_alter add (b1 int);
SyntaxException: line 1:27 no viable alternative at input '('
cqlsh:keyspace1> alter table test_alter add (b1 int, b2 int);
SyntaxException: line 1:27 no viable alternative at input '('
```

**Comment by lauranovich** [comment]
adding an issue for docs and will reference here as well

**Comment by slivne** [comment]
this is fixed in 3.1 closing the issue
